src/kubernetes.cc

```diff
@@ -66,6 +66,13 @@ constexpr const char kDockerIdPrefix[] = "docker://";
 
 constexpr const char kServiceAccountDirectory[] =
     "/var/run/secrets/kubernetes.io/serviceaccount";
+constexpr const char kKubernetesSchemaNameFormat[] =
+    "//container.googleapis.com/resourceTypes/{{type}}/versions/{{version}}";
```

Comment:
This is what the Metadata API currently supports - so out of the scope of this PR.

I agree that a prefix of k8s.io makes more sense, since these schemas will apply to K8s resources outside GCP as well. Will send them a request.